generated/nirfmxwcdma/nirfmxwcdma.proto

```diff
@@ -13,53 +13,3435 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxWCDMA";
+  int32 actual_array_size = 7;
+}
+
+message SEMFetchMeasurementStatusRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message SEMFetchMeasurementStatusResponse {
+  int32 status = 1;
+  SemMeasurementStatus measurement_status = 2;
+  int32 measurement_status_raw = 3;
+}
+
+message SEMFetchSpectrumRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message SEMFetchSpectrumResponse {
+  int32 status = 1;
+  double x0 = 2;
+  double dx = 3;
+  repeated float spectrum = 4;
+  repeated float relative_mask = 5;
+  repeated float absolute_mask = 6;
+  int32 actual_array_size = 7;
+}
+
+message SEMFetchTotalCarrierPowerRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message SEMFetchTotalCarrierPowerResponse {
+  int32 status = 1;
+  double total_carrier_power = 2;
+}
+
+message SEMFetchUpperOffsetMarginRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message SEMFetchUpperOffsetMarginResponse {
+  int32 status = 1;
+  SemUpperOffsetMeasurementStatus measurement_status = 2;
+  int32 measurement_status_raw = 3;
+  double margin = 4;
+  double margin_frequency = 5;
+  double margin_absolute_power = 6;
+  double margin_relative_power = 7;
+}
+
+message SEMFetchUpperOffsetMarginArrayRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message SEMFetchUpperOffsetMarginArrayResponse {
+  int32 status = 1;
+  repeated SemUpperOffsetMeasurementStatus measurement_status = 2;
+  repeated int32 measurement_status_raw = 3;
+  repeated double margin = 4;
+  repeated double margin_frequency = 5;
+  repeated double margin_absolute_power = 6;
+  repeated double margin_relative_power = 7;
+  int32 actual_array_size = 8;
+}
+
+message SEMFetchUpperOffsetPowerRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message SEMFetchUpperOffsetPowerResponse {
+  int32 status = 1;
+  double absolute_integrated_power = 2;
+  double relative_integrated_power = 3;
+  double absolute_peak_power = 4;
+  double peak_frequency = 5;
+  double relative_peak_power = 6;
+}
+
+message SEMFetchUpperOffsetPowerArrayRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message SEMFetchUpperOffsetPowerArrayResponse {
+  int32 status = 1;
+  repeated double absolute_integrated_power = 2;
+  repeated double relative_integrated_power = 3;
+  repeated double absolute_peak_power = 4;
+  repeated double peak_frequency = 5;
+  repeated double relative_peak_power = 6;
+  int32 actual_array_size = 7;
+}
+
+message SelectMeasurementsRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof measurements_enum {
+    MeasurementTypes measurements = 3;
```

Comment:
This doesn't have to be fixed now but we should add a note to circle back on it in the Beautify work item. I noticed that the `MeasurementTypes` enum looks like a bitfield. I think we should see if it makes sense to mark the params taking a `MeasurementTypes` enum as `bitfield_as_enum_array`. If it makes sense marking it that way will make it easier to pass a combo of the flags into the Request instead of having to construct the combination in `measurements_raw`.

Reply:
I will create a new PR for this issue. I need to generate the metadata from srcapigen repo again.